frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_permissions.py

```diff
@@ -11,6 +11,20 @@
 from . import GET, PUT, editable_copr, get_copr
 
 
+@apiv3_ns.route("/project/permissions/can_build_in/<ownername>/<projectname>")
```

Comment:
Perhaps a small thing: When I see `can_build_in`, I don't know who.  From the method name I'd expect a username as an argument.  Should we have a better name?

Reply:
Interesting perspective. I thought about it as "Can I (as a logged user) build in <ownername>/<projectname>". But you are right, why limitting this to _myself_ only. 

Maybe we can remove the `@api_login_required` decorator making this route available for everyone and instead, adding a `username` parameter?

Reply:
What I want to change is the behavior of `ProjectProxy.can_build_in` method. Either, I want to rename it to `assert_can_build_in` changing it to return a boolean.

But I will take a look what should we return in the API call as well.